regex.c

```diff
@@ -17,16 +17,19 @@ static bool *regexconfigured = NULL;
 static char **regexbuffer = NULL;
 static whitelistStruct whitelist = { 0, NULL };
 
+static FILE *checklist = NULL;
+
 static void log_regex_error(char *where, int errcode, int index)
 {
 	// Regex failed for some reason (probably user syntax error)
 	// Get error string and log it
 	size_t length = regerror(errcode, &regex[index], NULL, 0);
 	char *buffer = calloc(length,sizeof(char));
 	(void) regerror (errcode, &regex[index], buffer, length);
-	logg("ERROR %s regex %i: %s (%i)", index, where, buffer, errcode);
+	logg("ERROR %s regex on line %i: %s (%i)", where, index+1, buffer, errcode);
+	if(checklist)
+		fprintf(checklist, "%s\n", buffer);
```

Comment:
where is it going to be printed then?

Reply:
(I mean, in the interface, in the dashboard)

Reply:
If you look closely at this pull request, you'll find there reference to the connected dashboard pull request: 
![screenshot at 2018-08-20 13-39-05](https://user-images.githubusercontent.com/16748619/44338529-6c9f1680-a47e-11e8-8a3f-905b735b7e68.png)

https://github.com/pi-hole/AdminLTE/pull/804